modules/hostdb.go

```diff
@@ -7,7 +7,7 @@ import (
 
 const (
 	// Denotes a host announcement in the Arbitrary Data section.
-	HostAnnouncementPrefix = "HostAnnouncement"
+	PrefixHostAnnouncement = "HostAnnouncement\n"
```

Comment:
The remainder of the announcement is binary-encoded. No one is going to be reading or writing it by hand.